css/properties/perspective.json

```diff
@@ -39,7 +39,13 @@
               {
                 "prefix": "-moz-",
                 "version_added": "10",
-                "version_removed": "preview"
+                "flags": [
```

Comment:
Same here, just remove `"version_removed": "preview"` instead, not add the flag.